e2e-tests/docker-compose.yml

```diff
@@ -9,6 +9,7 @@ services:
       JDK_JAVA_OPTIONS: -Xmx512M
       KAFKA_ADDRESS: kafka:9092
       SCHEMA_REGISTRY_URL: http://schemaregistry:8081
+      USAGE_REPORTS_FINGERPRINT: "e2e-tests"
```

Comment:
isn't it better to disable usage statistics in tests at all ?

Reply:
disabled